script.js

```diff
@@ -0,0 +1,131 @@
+const menu = document.querySelector('.menu');
+const ham = document.querySelector('.ham');
+const xIcon = document.querySelector('.xIcon');
+const menuIcon = document.querySelector('.menuIcon');
+const overlay = document.getElementById('overlay');
+
+const toggleMenu = () => {
+  if (menu.classList.contains('showMenu')) {
+    menu.classList.remove('showMenu');
+    xIcon.style.display = 'none';
+    menuIcon.style.display = 'block';
+  } else {
+    menu.classList.add('showMenu');
+    xIcon.style.display = 'block';
+    menuIcon.style.display = 'none';
+  }
+};
+
+function openModal(modal) {
+  if (modal == null) return;
+  modal.classList.add('active');
+  overlay.classList.add('active');
+}
+
+function closeModal(modal) {
+  if (modal == null) return;
+  modal.classList.remove('active');
+  overlay.classList.remove('active');
+}
+
+ham.addEventListener('click', toggleMenu);
+
+const menuLinks = document.querySelectorAll('.menuLink');
+
+menuLinks.forEach(
+  (menuLink) => {
+    menuLink.addEventListener('click', toggleMenu);
+  },
+);
+
+const openModalButtons = document.querySelectorAll('[data-modal-target]');
+const closeModalButtons = document.querySelectorAll('[data-close-button]');
+
+openModalButtons.forEach((button) => {
+  button.addEventListener('click', () => {
+    const modal = document.querySelector(button.dataset.modalTarget);
+    openModal(modal);
+  });
+});
+
+overlay.addEventListener('click', () => {
+  const modals = document.querySelectorAll('.modal.active');
+  modals.forEach((modal) => {
+    closeModal(modal);
+  });
+});
+
+closeModalButtons.forEach((button) => {
+  button.addEventListener('click', () => {
+    const modal = button.closest('.modal');
+    closeModal(modal);
+  });
+});
+
+const modalObj = {
```

Comment:
Please follow the design for desktop and mobile exactly. The pop-up should have technologies and buttons on right and test on the left. The top section should have proper space between the text which can be seen in the design. Pop-up does not have a vertical scroll bar in the design, so please do not add a scroll bar on desktop view. Moreover, buttons are missing icons for all the views. Kindly see the screenshots below and fix all the design issues and make sure that it matches with the original design.
![popup-desktop](https://user-images.githubusercontent.com/35504149/132849389-9cfbe10a-c360-48ca-a97b-9a3d30f8e2a5.png)
![popup-top-desktop](https://user-images.githubusercontent.com/35504149/132849393-676ef38b-ad31-4788-8af9-6c63c630df3f.png)